internal/stack/shellinit.go

```diff
@@ -11,8 +11,7 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/elastic/go-sysinfo"
-	"github.com/elastic/go-sysinfo/types"
+	"github.com/shirou/gopsutil/process"
```

Comment:
To remove the `+incompatible` version in go.mod of this package, this import could be changed to:

```suggestion
	"github.com/shirou/gopsutil/v3/process"
```
I think it comes from because there is no go.mod in [v3.21.11 version](https://github.com/shirou/gopsutil/tree/v3.21.11)

Changin this, that suffix would be removed in go.mod (and there are also added a couple of indirect deps):
```diff
        github.com/olekukonko/tablewriter v0.0.5
        github.com/pkg/errors v0.9.1
        github.com/pmezard/go-difflib v1.0.0
-       github.com/shirou/gopsutil v3.21.11+incompatible
+       github.com/shirou/gopsutil/v3 v3.22.11
        github.com/spf13/cobra v1.6.1
        github.com/stretchr/testify v1.8.1
        golang.org/x/oauth2 v0.3.0
```
Just did a quick test for bash and it looks like it worked too:
```bash
 $ make build
go build -ldflags "-X github.com/elastic/elastic-package/internal/version.CommitHash=`git describe --always --long --dirty` -X github.com/elastic/elastic-package/internal/version.BuildTime=`date +%s` -X github.com/elastic/elastic-package/internal/version.Tag=`(git describe --exact-match --tags 2>/dev/null || echo '') | tr -d '\n'`" -o elastic-package

 $ ./elastic-package stack shellinit
Detected shell: bash
export ELASTIC_PACKAGE_ELASTICSEARCH_HOST=https://127.0.0.1:9200
export ELASTIC_PACKAGE_ELASTICSEARCH_USERNAME=elastic
export ELASTIC_PACKAGE_ELASTICSEARCH_PASSWORD=changeme
export ELASTIC_PACKAGE_KIBANA_HOST=https://127.0.0.1:5601
export ELASTIC_PACKAGE_CA_CERT=/home/mariorodriguez/.elastic-package/profiles/default/certs/ca-cert.pem
```

Reply:
Ah, good one, thanks!